src/WebJobs.Script/Diagnostics/MetricEventNames.cs

```diff
@@ -25,6 +25,8 @@ public static class MetricEventNames
 
         // language worker level events
         public const string WorkerInitializeLatency = "host.startup.outofproc.{0}worker.initialize.attempt{1}.latency";
+        public const string WorkerInvoked = "{0}worker.invoke";
```

Comment:
I'm thinking you should have a pair of worker.invoke.succeeded/worker.invoke.failed events similar to what we do for function invocations here https://github.com/Azure/azure-functions-host/blob/7054c51ad9666d6ae0f30da57054d9d8c457789b/src/WebJobs.Script.WebHost/Diagnostics/FunctionInstanceLogger.cs#L103.

That allows you to query the exact number of failures/successes in a given time window. With the events you have now, in a time window you would be incorrectly interpreting invocations which may be in progress but not completed as failures, right?

Reply:
Done